mathics/builtin/atomic/numbers.py

```diff
@@ -60,7 +60,9 @@
 
 @lru_cache()
 def log_n_b(py_n, py_b) -> int:
-    return int(mpmath.ceil(mpmath.log(py_n, py_b))) if py_n != 0 and py_n != 1 else 1
+    return (
```

Comment:
This could be moved to mathics.eval.numbers

Reply:
Let me elaborate a little on @mmatera' s comment.   Right now, we have a tree evaluator for Mathics3, and down the line if we are to get better performance, we will want an instruction interpreter. 

Moving evaluation routines somewhere under `mathics.eval`  (and here it would be `mahics.eval.atomic.numbers`) will help out when such large task is done. 

We are moving towards an organization where what is in `mathics.builtin` are the classes that implement the Builtin functions and inside their `eval()` routines we have largely the builtin properties (rules, attributes, doctest examples, function call matching signature, etc.)  and some parameter checking.  The non OO stuff and the bulk of the evaluation then is moved into `mathics.eval`, unless it is about fundamental Mathics objects (Strings, Integer, Patterns, etc.) which would go into  `mathics.core`.

Reply:
Perhaps, in the interests of not making this PR any bigger than it already is, this could be left to another PR?

Reply:
Ok, let's do it later